analysis/tools/ccsh/src/main/kotlin/de/maibornwolff/codecharta/tools/ccsh/parser/repository/IParserRepository.kt

```diff
@@ -0,0 +1,11 @@
+package de.maibornwolff.codecharta.tools.ccsh.parser.repository
+
+import de.maibornwolff.codecharta.tools.interactiveparser.InteractiveParser
+interface IParserRepository<T> {
```

Comment:
As far as I know, we do not prefix our interfaces with `I`. As such, I would stick to the current approach and remove it.